README.md

```diff
@@ -1,2 +1,33 @@
-# chef-bluematador
-Chef cookbook to install the Blue Matador agent.
+# bluematador_agent
+
+Installs/Configures the Blue Matador Agent.
+
+## Requirements
+
+ * chef >= 12.1
+
+## Platforms
+
+ * Amazon Linux
+ * CentOS
+ * Debian
+ * RedHat
+ * Ubuntu
+
+## Usage
+
+ 1. If you are using [https://docs.chef.io/berkshelf.html](Berkshelf), add the `bluematador_agent` cookbook to your Berksfile. You can also install the cookbook in to your repository using knife `knife cookbook site install bluematador_agent` or clone this repo.
+ 1. Set the account_id and api_key attributes.
+    ```
+    node['bluematador_agent']['api_key'] = '<your api key>'
```

Comment:
they should be adding this to the environment file, preferrably, not the individual nodes. Is that obvious enough?

Reply:
I think anyone using chef knows what this means, this is just an example (very similar to other chef install examples I found)